13/umbraco-cms/tutorials/add-google-authentication.md

```diff
@@ -114,41 +110,31 @@ If you are working with a Cloud project, see the [Working locally](https://docs.
 
 ### Installing a Nuget Package
 
-You can install and manage packages in Visual Studio either using the Package Manager Console (PowerShell) or the NuGet Package Manager.
-
-#### Option 1: Package Manager Console (PowerShell)
+You can install and manage packages in a project.
 
-The NuGet Package Manager Console lets you use NuGet PowerShell commands to manage NuGet packages on your project. You can use this option if you are comfortable using the Package Manager Console (PowerShell). The command listed below is specific to the Package Manager Console in Visual Studio:
+1. Navigate to your project/solution folder.
+2. Open a command prompt of your choice.
+3. Run the following command to install the `Microsoft.AspNetCore.Authentication.Google` package.
 
-1. Open your project/solution in Visual Studio.
-2. Go to **Tools** > **NuGet Package Manager** > **Package Manager Console**.
+    ```cli
+    dotnet add package Microsoft.AspNetCore.Authentication.Google --version 8.0.3
```

Comment:
just tested and it works without mentioning the version. If it is like I am suggesting here, it will always install the latest version. you can specify the version if you want to install older versions. Perhaps this can be changed in the info note below?
```suggestion
    dotnet add package Microsoft.AspNetCore.Authentication.Google
```